config/plugin/upstream_dns_resolver.sh

```diff
@@ -6,9 +6,11 @@ UNKNOWN=2
 
 readonly fqdn_a_record="$1"
 
+
+
 # We send a single attempt only, without retries and with 1s timeout.
 # We expect to receive an IPv4 address for a given A record.
-dig_cmd_out="$(dig -t A @169.254.20.10 +tries=1 +retry=0 +time=1 +short "${fqdn_a_record}" 2>&1 | head -n1)"
+dig_cmd_out="$(dig -t A @${LOCAL_DNS_RESOLVER_IP} +tries=1 +retry=0 +time=1 +short "${fqdn_a_record}" 2>&1 | head -n1)"
```

Comment:
Do you know where the value for `LOCAL_DNS_RESOLVER_IP` comes from?

Reply:
from here: https://github.com/uswitch/cloud-kubernetes-extensions/pull/1765/files#diff-7699db3e61be327566375366dd640f6c66c14f354d404999b1068c5f97ce4d05R17